README.md

```diff
@@ -73,6 +87,33 @@ CVN-123 # Mitigated the risk of this since we only use one method in this packag
 CVN-543
 ``` 
 
+### Nexus IQ Server Options
+
+By default, assuming you have an out of the box Nexus IQ Server running, you can run `nancy` like so:
+
+`go list -m all | ./nancy iq -application public-application-id`
+
+It is STRONGLY suggested that you do not do this, and we will warn you on output if you are.
+
+A more logical use of `nancy` against Nexus IQ Server will look like so:
+
+`go list -m all | ./nancy iq -application public-application-id -user nondefaultuser -token yourtoken -server-url http://adifferentserverurl:port -stage develop`
+
+Options for stage are as follows:
+
+`build, develop, stage-release, release`
+
+By default `-stage` will be `develop`.
```

Comment:
Why would the default be `develop`?? Im going to guess 99% of the time this will be running on CI so shouldn't it be `build`. Keep in mind i have no real idea what this stages mean in IQ (only briefly read some of the docs :) ) so I'll defer to you of course.

Reply:
You have `build` > `develop` > `release-stage` > `release`. I went with develop as a default as Nancy can be used by developers locally pretty easy. We could set it to `build` for sure, but it kinda depends on how you setup your policy TBH with Nexus IQ. I originally had it as `build` ftr. I can go either way!

Reply:
Cool cool.... that makes sense then. 

#scopeCreepAndMaybeNotWorthIt
But would it make sense to detect if on CI (looking for TRAVIS, CIRCLECI, CI, etc etc) environment variables and if one of those is set you change the stage automagically to `build`/`release-stage`?? Maybe that is still back into the "how you setup your policy in Nexus IQ" again and it doesn't matter. ¯\_(ツ)_/¯

Reply:
I LIKE THAT. We'd still need to allow someone to override it, though, because people use Travis, CircleCI etc... to do CD too.

Reply:
Yeah for sure...if you pass it in then that is the value to be used. But nancy will attempt to "smart default" it for you.

Reply:
Nancy is so clever!

Reply:
I think we should create a ticket for this, it was a fantastic suggestion, however I dunno if I want to do it in this PR.